required/latex-lab/latex-lab-sec.dtx

```diff
@@ -68,7 +68,9 @@
 % \begin{itemize}
 %  \item The heading/title text of the section should be surrounded by a 
 %  heading tag, typically \texttt{Hn} with some value of \texttt{n}.
-%  The number of the section command can optionally be put in a \texttt{Lbl}. 
+%  Theoretically the number of the section command could be 
+%  optionally be put in a \texttt{Lbl}. But AT doesn't handle this well, so 
```

Comment:
In theory, one could put the number of the section command in an \texttt{Lbl}. However, current AT doesn't ...